src/components/ContactDetails.vue

```diff
@@ -854,8 +854,12 @@ export default {
 		 * Save the contact. This handler is triggered by the save button.
 		 */
 		async onSave() {
-			await this.updateContact()
-			this.editMode = false
+			try {
+				await this.updateContact()
+				this.editMode = false
+			} catch (error) {
+				showError(t('contacts', 'Unable to update contact'))
```

Comment:
the error will go silent now. could you add an error.log in a follow-up pr? :)

Reply:
it's logged in contacts.js then thrown again